src/Core/Checkout/Document/SalesChannel/DocumentRoute.php

```diff
@@ -59,12 +65,17 @@ public function download(
             throw DocumentException::customerNotLoggedIn();
         }
 
+        $fileType = $request->query->get('fileType') ?? $fileType;
```

Comment:
`{{baseurl}}/store-api/document/download/documentId/deepLinkCode?fileType=` currently returns an empty response instead of falling back to `pdf`

```suggestion
        $fileType = $request->query->get('fileType') ?: $fileType;
```

or go with https://github.com/shopware/shopware/pull/12944#discussion_r2549618013

Reply:
we use the accept header now